programs/programs/nc/nc_head_start/tests.py

```
@@ -0,0 +1,632 @@
+        - 30% of gross income: $3,500 × 0.3 = $1,050/month
+        - Rent: $2,000/month exceeds $1,050 by $950/month
+        - Housing adjustment: $950 × 12 = $11,400/year
+        - Adjusted income: $42,000 - $11,400 = $30,600 (114% FPL) - ELIGIBLE!
+
+        Eligible children:
+        - Person 2 (age 6): NOT eligible (school-age, no disability)
+        - Person 3 (age 3): ELIGIBLE (preschool age)
+
+        Value calculation:
+        - Only Person 3 is eligible: preschool rate = $1,000/month
+        - Estimated savings: $1,000 × 12 = $12,000/year
```

Ruff (0.14.8)

Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)